pubspec.yaml

```diff
@@ -77,8 +77,8 @@ dependencies:
     # path: /Users/nt4f04und/dev/prj/nt4f04unds_widgets
 
     git:
-      url: https://github.com/nt4f04uNd/nt4f04unds_widgets
-      ref: 5.0.0
+      url: https://github.com/Abestanis/nt4f04unds_widgets
```

Comment:
TODO: Replace this back once https://github.com/nt4f04uNd/nt4f04unds_widgets/pull/27 is merged.